manifests/cf-manifest/operations.d/020-bosh-set-stemcells.yml

```diff
@@ -5,7 +5,7 @@
   value:
     - alias: "3468"
       os: ubuntu-trusty
-      version: "3468.42"
+      version: "3468.51"
     - alias: default
       os: ubuntu-trusty
-      version: "3586.16"
+      version: "3586.23"
```

Comment:
Actually we need 3586.24 for https://www.cloudfoundry.org/blog/usn-3686-1/ and https://www.cloudfoundry.org/blog/usn-3684-1/

I will update that